src/test/java/org/opensearch/searchrelevance/action/judgment/JudgmentsIT.java

```diff
@@ -32,6 +32,64 @@
 @OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.SUITE)
 public class JudgmentsIT extends BaseSearchRelevanceIT {
 
+    @SuppressWarnings("unchecked")
+    private List<String> extractStatuses(Map<String, Object> result) {
+        Map<String, Object> hitsObj = (Map<String, Object>) result.get("hits");
```

Comment:
funny how I see this pattern all over the place!  In our front end code, our tests, etc...  Just a comment!